test/test_sanity.py

```diff
@@ -794,7 +794,7 @@ def test_embuilder_external_ports_options(self):
   def test_binaryen_version(self):
     restore_and_set_up()
     with open(EM_CONFIG, 'a') as f:
-      f.write('\nBINARYEN_ROOT = "' + self.in_dir('fake') + '"')
+      f.write('\nBINARYEN_ROOT = "' + self.in_dir('fake').replace('\\', '/') + '"')
```

Comment:
Can you do add an `r` to the start on the string here instead?   e.g. `f.write('\nBINARYEN_ROOT = r"' + self.in_dir('fake') + '"')`